src/api/neighborhoods/provider.tsx

```diff
@@ -48,3 +48,21 @@ export function NeighborhoodProvider({ children }: { children: ReactNode }) {
     </NeighborhoodContext.Provider>
   );
 }
+
+export async function getServices(
```

Comment:
If we find a function like this useful in other places I could see it getting added to ManyJS.

Reply:
Yeah, this definitely looks useful in other contexts to discover what's available!